src/components/last-planner/last-planner-view.tsx

```diff
@@ -0,0 +1,258 @@
+import { useApi } from "hooks/use-api";
+import { Fragment, useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import { TaskWithInterval } from "types";
+import { getContrastForegroundColor, hexFromString } from "utils";
+import { splitIntervalByDuration } from "utils/date-time-utils";
+import {
+  distributeOverlappingTasksToRows,
+  fillGapsBetweenTasksInRow,
+  getTimelineIntervalByTasks,
+  groupTasksByOverlap,
+  mapTasksAndUsersByUserId,
+  sortTasksByStartTime,
+} from "utils/last-planner-utils";
+import { TaskRowCell } from "./task-row-cell";
+
+const LastPlannerTableWrapper = styled("div")(({ theme }) => ({
+  position: "relative",
+  paddingBottom: theme.spacing(2),
+  "& table": {
+    width: "100%",
+    borderCollapse: "separate",
+    borderSpacing: 0,
+    "& th": {
+      borderTop: `1px solid ${theme.palette.grey[300]}`,
+      borderBottom: `1px solid ${theme.palette.grey[300]}`,
+      borderRight: `1px solid ${theme.palette.grey[300]}`,
+    },
+    "& td": {
+      borderBottom: `1px solid ${theme.palette.grey[300]}`,
+      borderRight: `1px solid ${theme.palette.grey[300]}`,
+    },
+    "& th:first-of-type, & td:first-of-type": {
+      borderLeft: `1px solid ${theme.palette.grey[300]}`,
+    },
+  },
+}));
+
+const FixedHeightTableRow = styled("tr")({ height: 50 });
+
+type StickyTableCellProps = {
+  top?: number;
+  left?: number;
+  colSpan?: number;
+  constrainWidth?: boolean;
+};
+
+const StickyTableCell = styled("td", {
+  shouldForwardProp: (prop) =>
+    prop !== "top" && prop !== "left" && prop !== "noLeftBorder" && prop !== "constrainWidth",
+})<StickyTableCellProps>(({ top, left, colSpan = 1, constrainWidth = false }) => ({
+  position: "sticky",
+  top: top,
+  left: left,
+  backgroundColor: "white",
+  minWidth: 40,
+  maxWidth: constrainWidth ? 40 * colSpan : undefined,
+  verticalAlign: "middle",
+  overflow: "hidden",
+  textOverflow: "ellipsis",
+  zIndex: 1,
+}));
+
+type Props = {
+  projectId: string;
+  editMode?: boolean;
+};
+
+const LastPlannerView = ({ projectId, editMode }: Props) => {
+  const navigate = useNavigate({ from: "/projects/$projectId/tasks" });
+  const { t } = useTranslation();
+  const { tasksApi } = useApi();
+  const queryClient = useQueryClient();
+
+  const listTasksQuery = useListTasksQuery({ projectId });
+  const tasks = listTasksQuery.data ?? [];
+  const listProjectUsersQuery = useListUsersQuery({ projectId });
+  const users = listProjectUsersQuery.data?.users ?? [];
+  const jobPositionsQuery = useListJobPositionsQuery({ max: 9999 });
+  const jobPositions = jobPositionsQuery.data?.jobPositions ?? [];
+
+  const updateTaskMutation = useMutation({
+    mutationFn: (task: Task) => tasksApi.updateTask({ projectId: projectId, taskId: task.id as string, task: task }),
+    onMutate: async (taskToUpdate) => {
+      const queryKey = ["projects", projectId, "tasks", {}];
```

Comment:
This is because query key array length matters. Empty object is different from leaving it out alltogether.
We want to speficically cancel queries and update query results for the query done in this specific view, which lists project tasks without any filters. This needs to be done by specifically providing an empty filter object to query key. After we have confirmed, that the query was successful, we can then proceed to invalidate the results for _all_ of the project task queries, so that we have up-to-date data everywhere.